h2o-core/src/main/java/water/MRTask.java

```diff
@@ -240,6 +240,7 @@ private Frame closeFrame(Key key, String[] names, String[][] domains, Futures fs
     }
     return new Frame(key,names,vecs);
   }
+  public void map(){}
```

Comment:
not needed anymore

Reply:
Removed in https://github.com/h2oai/h2o-3/pull/3577/commits/70b82635533f9972649766784ef7d404d459a6ed